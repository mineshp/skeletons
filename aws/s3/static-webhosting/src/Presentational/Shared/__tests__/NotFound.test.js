import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../NotFound';

describe('Presentational > NotFound', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(
      <NotFound />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
