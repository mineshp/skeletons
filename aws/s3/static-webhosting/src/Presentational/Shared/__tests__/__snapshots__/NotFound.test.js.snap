// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Presentational > NotFound Renders correctly 1`] = `
<div>
  <Container
    text={true}
  >
    <Header
      as="h2"
    >
      404 Page Not Found!
    </Header>
    <p>
      Oops it looks like you have ventured off-course.
    </p>
  </Container>
</div>
`;
